frontend/src/components/features/conversation-panel/conversation-card.tsx

```diff
@@ -112,9 +113,10 @@ export function ConversationCard({
     // Fetch the VS Code URL from the API
     if (conversationId) {
       try {
-        const response = await fetch(
-          `/api/conversations/${conversationId}/vscode-url`,
-        );
+        const baseUrl =
+          OpenHands.getConversationUrl() ||
+          `/api/conversations/${conversationId}`;
```

Comment:
Similar to other comment:

```suggestion
        const baseUrl = getConversationUrl(conversationId);
```